src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 spring.application.name=d2l-mock
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
we already have an `application-dev.yml` file saved that gets the DB working. I dont think we need to merge this one too

Reply:
No prob.